templates/csharp/xplat/AvaloniaTest.Browser/Program.cs

```diff
@@ -8,9 +9,8 @@
 
 internal partial class Program
 {
-    private static void Main(string[] args) => BuildAvaloniaApp()
-        .UseReactiveUI()
-        .SetupBrowserApp("out");
+    private static async Task Main(string[] args) => await BuildAvaloniaApp()
+            .StartBrowserAppAsync("out");
```

Comment:
I don't remember, do we use or not reactive on the xplat template? It was removed from here but not from f#.

Reply:
xplat still uses ReactiveUI (only) for MVVM, that removal was a mistake, will add it again